File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/SavedAlbum.java
package com.timothy.spotifyarchitecture.retrofit.models;

import android.os.Parcel;
import android.os.Parcelable;

import com.timothy.spotifyarchitecture.retrofit.models.Album;

/**
 * <a href="https://developer.spotify.com/web-api/object-model/#saved-album-object">Saved album object model</a>
 */
public class SavedAlbum {
    public String added_at;
    public Album album;

    public SavedAlbum() {
    }

}

File: spotify-web/src/main/kotlin/com/timmahh/spotifyweb/retrofit/models/SavedAlbum.kt
package com.timmahh.spotifyweb.retrofit.models

/**
 * [Saved album object model](https://developer.spotify.com/web-api/object-model/#saved-album-object)
 */
class SavedAlbum {
    var added_at: String = ""
    var album: Album = Album()
}